### PR TITLE
all-board-api-mock added with BoardController, BoardDTO and BoardService

### PR DESCRIPTION
**Pull request by Radenzvous**
- Added BoardController, BoardDTO and BoardService
- Mocking 2 hardcoded objects
- GET /boards will show the objects

### CODE REVIEW

**Review by MaciejJ0 — APPROVED** [review]

**Review by Szachista — APPROVED** [review]

File: src/main/java/com/intive/patronage22/szczecin/retroboard/dto/BoardDTO.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Value;
+
+
+@Value
+@Getter
+@RequiredArgsConstructor
+public class BoardDTO {
+    public enum State {
```
Comment (patrykzawadzkiintive) [inline src/main/java/com/intive/patronage22/szczecin/retroboard/dto/BoardDTO.java:13]: Please move that enum to separate enum class.

File: src/main/java/com/intive/patronage22/szczecin/retroboard/dto/BoardDTO.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Value;
+
+
+@Value
+@Getter
+@RequiredArgsConstructor
+public class BoardDTO {
```
Comment (patrykzawadzkiintive) [inline src/main/java/com/intive/patronage22/szczecin/retroboard/dto/BoardDTO.java:12]: Rename class to BoardDto -> lets follow camelCase naming system.

File: src/main/java/com/intive/patronage22/szczecin/retroboard/service/BoardService.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
+package com.intive.patronage22.szczecin.retroboard.service;
```
Comment (patrykzawadzkiintive) [inline src/main/java/com/intive/patronage22/szczecin/retroboard/service/BoardService.java:1]: Something is wrong with class formatting. Please also do not leave so many empty spaces. In intelij open class click on top bar Code -> Reformat Code. It should reformat it automatically.

File: src/main/java/com/intive/patronage22/szczecin/retroboard/controller/BoardController.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/boards")
+public class BoardController {
+
+
+private final BoardService boardService;
+
+@GetMapping
```
Comment (patrykzawadzkiintive) [inline src/main/java/com/intive/patronage22/szczecin/retroboard/controller/BoardController.java:25]: Classs formatting.

File: src/main/java/com/intive/patronage22/szczecin/retroboard/RetroboardApplication.java [hunk 5]
```diff
@@ -9,5 +9,4 @@
 	public static void main(String[] args) {
 		SpringApplication.run(RetroboardApplication.class, args);
 	}
-
```
Comment (patrykzawadzkiintive) [inline src/main/java/com/intive/patronage22/szczecin/retroboard/RetroboardApplication.java:12]: Unnecessary change.

**Review by patrykzawadzkiintive — CHANGES_REQUESTED** [review]

**Comment by patrykzawadzkiintive** [comment]
Please also add unit test and/or integration test for Controller class.

**Review by patrykzawadzkiintive — APPROVED** [review]

**Review by golldfish — APPROVED** [review]

**Pull request merged**